packages/cmrjs/tests/cmr-utils/test-cmr-utils.js

```diff
@@ -729,3 +730,67 @@ test(
   'metadataObjectFromCMRFile throws PreconditionFailed when ETag does not match CMR UMMG JSON metadata file',
   testMetadataObjectFromCMRFile('s3://bucket/fake.cmr.json')
 );
+
+test.serial('publish2CMR passes cmrRevisionId to publishECHO10XML2CMR', async (t) => {
+  const cmrFileObject = { filename: 'test.cmr.xml', granuleId: 'testGranuleId' };
+  const updatedXmlFile = { ...cmrFileObject, metadataObject: {} };
+
+  const publishECHO10XML2CMRSpy = sinon.spy(() => Promise.resolve());
+  const revertPublishECHO10XML2CMRSpy = cmrUtil.__set__('publishECHO10XML2CMR', publishECHO10XML2CMRSpy);
+  const credentials = await getCmrSettings();
+  const cmrRevisionId = Math.random;
```

Comment:
Sorry, I should have been more clear. `Math.random` isn't a function call, it's just a reference to a function of the `Math` global.

You would need to use: `Math.floor(Math.random() * 100)`

Reply:
Oh, that makes sense.